include/core/HY_Features_MPI.hpp

```diff
@@ -11,84 +11,29 @@
 #include <network.hpp>
 #include <Formulation_Manager.hpp>
 #include <Partition_Parser.hpp>
+#include <HY_Features.hpp>
 
 namespace hy_features {
 
-    class HY_Features_MPI {
+    class HY_Features_MPI: public HY_Features {
       public:
       
       using Formulation_Manager = realization::Formulation_Manager;
       
         HY_Features_MPI(PartitionData partition_data, geojson::GeoJSON linked_hydro_fabric,
                         std::shared_ptr<Formulation_Manager> formulations, int mpi_rank, int mpi_num_procs);
 
-        std::shared_ptr<HY_CatchmentRealization> catchment_at(std::string id) {
-            return (_catchments.find(id) != _catchments.end()) ? _catchments[id]->realization : nullptr;
-        }
-
-        inline auto catchments() {
-            return network.filter("cat");
-        }
+        HY_Features_MPI() = delete; //Explicitly delete the default constructor
```

Comment:
```suggestion
        HY_Features_MPI() = delete;
```
Comment's just repeating the plain syntax

Reply:
Note to self, remove note to self!